app/views/profiles/edit_publications.html.erb

```diff
@@ -47,6 +48,16 @@
           <td class="oa-status">
             <%= fa_icon(a.open_access_status_icon) %>
           </td>
+          <td>
+            <% if a.user.orcid_access_token && a.publication.orcid_allowed? %>
+              <% if a.orcid_resource_identifier.present? %>
+                <sup>*</sup>This information has been added to your ORCID record.
+              <% else %>
+                <%= button_to "Add to my ORCID Record", orcid_works_path, class: 'btn orcid-button',
+                              remote: false, params: { authorship_id: a.id } %>
+              <% end %>
+            <% end %>
+          </td>
```

Comment:
This looks good. As we're adding controls to the UI and/or changing the workflow, we should also just keep in mind that at some point we'll probably want to give users the ability to update and delete works that they've added to orcid.